3.analysis/scripts/3.t_tests.py

```diff
@@ -258,7 +258,7 @@ def generate_latent_df(latent_df, category_ids):
 t_test_diff_adult_vs_ped = ttest_ind(adult_LC_latent_float_df, ped_NB_latent_float_df)
 t_test_diff_adult_vs_ped = pd.DataFrame(t_test_diff_adult_vs_ped).T
 t_test_diff_adult_vs_ped.columns = ["t_stat", "p_value"]
-t_test_diff_adult_vs_ped['comparison'] = 'Adult vs Pediatric'
+t_test_diff_adult_vs_ped['comparison'] = 'Lung Cancer vs Neuroblastoma'
```

Comment:
These were chosen previously by a different contributor, but were chosen because they have the largest sample sizes for comparison. Would definitely be interesting to compare similar cancers as well!